### PR TITLE
initial code to display funding link on podcast page

### PR DESCRIPTION
**Pull request by eurbasius**

### CODE REVIEW

**Comment by RyanHirsch** [comment]
You should rebase off of `develop` if possible. That will pull down the correct prettier config as well as create an action to verify lint.

At the very least, you need the `.prettierrc` from #344.

File: pages/episode/[episodeId].tsx [hunk 1]
```diff
@@ -102,6 +108,13 @@ export default function Episode({
     }
   }
 
+  if (serverEpisode.funding && serverEpisode.podcast.funding) {
+    const combinedFundingLinks = serverEpisode.funding.concat(serverEpisode.podcast.funding)
+    fundingLinks = combinedFundingLinks.map((link, index) => {
+      return <FundingLink key={index} link={link.url} value={link.value}></FundingLink>
```
Comment (RyanHirsch) [inline pages/episode/[episodeId].tsx:114]: You should generally avoid using indexes as keys, in this circumstance, I think `link.url` will be unique in this group and should be used as the key.

File: pages/podcast/[podcastId].tsx [hunk 2]
```diff
@@ -121,6 +122,13 @@ export default function Podcast({
     title: podcastTitle
   }
 
+  let fundingLinks
+
+  if (serverPodcast.funding.length) {
+    fundingLinks = serverPodcast.funding.map((link, index) => {
+      return <FundingLink key={index} link={link.url} value={link.value}></FundingLink>
```
Comment (RyanHirsch) [inline pages/podcast/[podcastId].tsx:129]: Same as previous comment, don't use indexes as keys

**Review by mitchdowney — APPROVED** [review]
Looks great! Thanks again :)

**Pull request merged**